packages/fxa-settings/src/components/LinkRememberPassword/index.tsx

```diff
@@ -14,15 +15,20 @@ const LinkRememberPassword = ({
   email,
   forceAuth = false,
 }: LinkRememberPasswordProps) => {
-  const linkTarget = `${forceAuth ? '/force_auth' : '/signin'}`;
-  let target = linkTarget;
+  const location = useLocation();
+  const params = new URLSearchParams(location.search);
```

Comment:
I kinda like this better!

Reply:
I think this approach can make sense for adding/deleting query params. For validating incoming and expected ones that get set on `integration.data`, we can use a query param model.